R/Transformations.R

```diff
@@ -491,8 +491,10 @@ TADA_CalculateTotalNP <- function(.data,
   
   # add rows back at end but do not include in TN/TP summation
   dat_addback <- dat[(dat$TADA.ResultValueAggregation.Flag %in%
-                        "Considered in max aggregation function but not selected"), ]
-  dat_addback$TADA.NutrientSummation.Flag <- "Not used to calculate Total N or P."
+                        c(paste0("Used in ", daily_agg, " aggregation function but not selected"))), ]
```

Comment:
Good catch. I am updating this in both functions for consistency to:

c(paste0("**Considered in** ", agg_fun, " aggregation function but not selected"))